### DISCUSSION TITLE
Obviously a problem with AIR and AMD Adrenalin 22.7.1 / Win 11

### DISCUSSION THREAD

**Discussion by ramraver** [airsdk/Adobe-Runtime-Support#2050] [General]
A client of us said he was not able to start our AIR app with a GPU AMD RX6800 and the driver 
Adrenalin 22.7.1 installed on a Windows 11 system. He sent a screenshot with just a Windows window of our app and nothing inside. He downgraded that driver and then it worked. Took some time to figure out where the reason was.

**Comment by ajwfrost** [comment]
Yes, we're seeing this with some of the new AMD/Radeon drivers. We've just created a temporary workaround so you could try limiting the app to using Direct3D 9, whilst we try to work out how to detect (or even fix) the problem with Direct3D 11..

https://github.com/airsdk/Adobe-Runtime-Support/issues/1963#issuecomment-1206060772

thanks

**Reply by ramraver** [reply]
> Yes, we're seeing this with some of the new AMD/Radeon drivers. We've just created a temporary workaround so you could try limiting the app to using Direct3D 9, whilst we try to work out how to detect (or even fix) the problem with Direct3D 11..
> 
> [#1963 (comment)](https://github.com/airsdk/Adobe-Runtime-Support/issues/1963#issuecomment-1206060772)
> 
> thanks

Andrew,
I would urgently like to know, if there are plans to support a (transparent) video format in Air?
BR
Kai

**Reply by ajwfrost** [reply]
Hi - we're looking to support additional formats in a more generic way, which would allow people to add their own capabilities and have them integrate to AIR into the display list (and via StageVideo, VideoTexture etc too). As a proof of concept of this we're looking at the NDI framework so basically it would mean we can create an ANE that links up with NDI, and we could then display videos from NDI sources within AIR.
In terms of built-in formats, a lot would depend on which platforms you're talking about .. we'd generally be dependent upon what capabilities are provided by the operating system now; in the past, Adobe had built-in software decoders for H.264/AAC etc but those are no longer available due to patent/licensing issues, so we just have Sorenson, VP6 and MP3 support built in, for the other formats it goes out to the OS..
thanks

**Reply by ajwfrost** [reply]
Was about to make this command also on #2113 and just realised that was one that you had kicked off.. -> will add the details there.

**Reply by ramraver** [reply]
Hi Andrew,
I guess NDI and an alpha channel supporting video format are strongly indicated and urgently needed. At least for us.

**Comment by ramraver** [comment]
Thanks Andrew.
